.changes/clone-stronghold.md

```diff
@@ -0,0 +1,5 @@
+---
+"iota-stronghold": minor
```

Comment:
I'd consider this to be a `patch`, but I'm fine with either.

Reply:
hmm a patch would be a bugfix, but making stronghold clonable extends it's functionality, even though it's minimal :)